l10n_es_ticketbai_api/demo/res_partner_demo.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Binovo IT Human Project SL
+     Copyright 2021 Landoo Sistemas de Informacion SL
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+    <record id="partner_tech" model="res.partner">
+        <field name="name">Tech User</field>
+        <field name="company_id" ref="base.main_company" />
+        <field name="email">tech@yourcompany.example.com</field>
+        <field name="company_name">Binovo IT Human Project S.L.</field>
+        <field name="street">Astigarraga bidea 2, 2ª Izquierda, Oficina 10-11</field>
+        <field name="city">Oiartzun</field>
+        <field name="zip">20180</field>
+        <field name="country_id" ref="base.es" />
+    </record>
+    <record id="user_tech" model="res.users">
+        <field name="partner_id" ref="partner_tech" />
+        <field name="login">tech</field>
+        <field name="password">tech</field>
+        <field name="signature" type="xml">
```

Comment:
Esta linea arroja warning en el log

2023-05-30 20:32:15,408 255 WARNING odoo odoo.tools.convert: HTML field 'signature' is declared as `type="xml"`